packages/router/test/bindings/messaging/auctionRequest.spec.ts

```diff
@@ -82,25 +82,53 @@ describe("auctionRequestBinding", () => {
     expect(messagingMock.publishAuctionResponse).to.be.calledOnceWith(from, inbox, { bid, bidSignature });
   });
 
+  it("should work if new auction happens in longer than 5s", async () => {
+    await new Promise((resolve) => {
```

Comment:
I think we should overwrite the lastAuctionMap in the tests instead of waiting 5 seconds for each test.

Reply:
Ok. I agree